README.md

```diff
@@ -227,16 +227,23 @@ Attribute | Values
 <CameraKitCameraScreen
     actions={{ rightButtonText: 'Done', leftButtonText: 'Cancel' }}
     onBottomButtonPressed={(event) => this.onBottomButtonPressed(event)}
-    scanBarcode={true}
-    laserColor={"blue"}
-    frameColor={"yellow"}
-
-    onReadQRCode={((event) => Alert.alert("Qr code found"))} //optional
-    hideControls={false}           //(default false) optional, hide buttons and additional controls on top and bottom of screen
-    showFrame={true}   //(default false) optional, show frame with transparent layer (qr code or barcode will be read on this area ONLY), start animation for scanner,that stoped when find any code. Frame always at center of the screen
-    offsetForScannerFrame = {10}   //(default 30) optional, offset from left and right side of the screen
-    heightForScannerFrame = {300}  //(default 200) optional, change height of the scanner frame
-    colorForScannerFrame = {'red'} //(default white) optional, change colot of the scanner frame
+    flashImages={{
+    on: require('./../images/flashOn.png'),
+    off: require('./../images/flashOff.png'),
+    auto: require('./../images/flashAuto.png')
+    }}
+    
+    showFrame={true} //(default false) optional, show frame with transparent layer (qr code or barcode will be read on this area ONLY), start animation for scanner,that stoped when find any code.
+    laserColor={"blue"} // optional (default is white) property that set color of the scanning line in scanning frame
+    surfaceColor={"black"} // optional (default is black) when scanner appears, we see smooth animation with fade from setted color
+    frameColor={"yellow"} // optional (default is white) property that set color of corners scanning frame
+    onReadCode={((event) => this.setState({ example: CheckingScreen }))} // callback usage , if you want to get value that was scanned use this for example onReadCode={((event) => Alert.alert(Qr code found ${event.nativeEvent.codeStringValue}))}
```

Comment:
Can you please show how to get the code string value here?

Reply:
But I had write it in comments if you want to get value that was scanned use this for example onReadCode={((event) => Alert.alert(Qr code found ${event.nativeEvent.codeStringValue}))}